### ISSUE TITLE
Cell Props

### ISSUE THREAD

**Issue by DmytroOSDB** [komarovalexander/ka-table#118]
Hello @komarovalexander 

I found that the interface `IHeadCellProps` isn't fully typed. These props are used in `childComponents -> headCell -> elementAttributes`

```
export interface IHeadCellProps {
    areAllRowsSelected: boolean;
    childComponents: ChildComponents;
    columnReordering?: boolean;
    columnResizing?: boolean;
    column: Column;
    dispatch: DispatchFunc;
    sortingMode: SortingMode;
}
```
![image](https://user-images.githubusercontent.com/77971524/106126559-2cbbdd80-6166-11eb-8b1b-057672ebb24b.png)

Many thanks, Dmytro

**Comment by komarovalexander** [comment]
Hi @DmytroOSDB 
Do you find these 3 options useful? 
They were added in the last minor release as a result of the spread operator https://github.com/komarovalexander/ka-table/pull/117/files#diff-64504a23f1f40c3c2919d1c7b8c7aef13fcca3e8dace734764d343267368f10bR19 
I think they should not be here and it should be fixed as a bug

**Comment by komarovalexander** [comment]
fixed in 6.5.1

**Comment by DmytroOSDB** [comment]
Thank you for the update!
